.travis.yml

```diff
@@ -16,3 +16,4 @@ env:
     global:
         - FACEBOOK_APP_ID=198798870326423
         - FACEBOOK_SECRET=2db4d76fe8a336cf292470c20a5a5684
+        - FACEBOOK_USER_ID=12345
```

Comment:
Agreed, though this feature wasn't there when we first talked about this :) ... happy to figure out how to make that change.